test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/Listener/RequestHeaderTests.cs

```diff
@@ -93,6 +93,56 @@ public class RequestHeaderTests
             }
         }
 
+        [ConditionalFact]
```

Comment:
You included Linq in your last PR: https://github.com/aspnet/HttpSysServer/pull/401/files#diff-2e2f490c4dbd4e72ad07b40f9a1d20bcR6. Was that a mistake or do you actually need it?

Reply:
Interesting. This PR didn't need it. I may have forgotten to clean it up in the last PR.